enterprise/app/trends/drilldown_page.tsx

```diff
@@ -431,6 +484,58 @@ export default class DrilldownPageComponent extends React.Component<Props, State
     return "To see drilldown charts and invocations, click and drag to select a region in the chart above";
   }
 
+  renderZoomChip(): React.ReactElement | null {
+    const zoomEligible = this.currentHeatmapSelection && this.currentHeatmapSelection.invocationsSelected > 1;
+    const zoomToSummarize = zoomEligible ? this.currentHeatmapSelection : this.currentZoomFilters;
+    if (!zoomToSummarize) {
+      return null;
+    }
+
+    const startDate = moment(zoomToSummarize.dateRangeMicros.startInclusive / 1000).format("YYYY-MM-DD");
+    const endDate = moment((zoomToSummarize.dateRangeMicros.endExclusive - 1) / 1000).format("YYYY-MM-DD");
+    const startValue = this.renderYBucketValue(zoomToSummarize.bucketRange.startInclusive);
+    const endValue = this.renderYBucketValue(zoomToSummarize.bucketRange.endExclusive);
+
+    return zoomEligible ? (
+      <div className="drilldown-page-zoom-summary">
+        <OutlinedButton
+          title="Zoom in on this selection"
+          className="drilldown-page-zoom-button"
+          onClick={() => this.handleHeatmapZoom(this.currentHeatmapSelection)}>
+          <ZoomIn className="icon"></ZoomIn>
+          <div className="drilldown-page-zoom-filters">
+            <div className="drilldown-page-zoom-filter-attr">
+              Date: {startDate} - {endDate}
+            </div>
+            <div className="drilldown-page-zoom-filter-attr">
+              Value: {startValue} - {endValue}
+            </div>
+          </div>
+        </OutlinedButton>
+      </div>
+    ) : (
+      <div className="drilldown-page-zoom-summary buttonless">
```

Comment:
nit: the "buttonless" is mildly confusing because the div contains a FilledButton :sweat_smile:

Reply:
ahahaha sorry, i restructured this like five times.  renamed it to 'zoomed', which is the actual state instead of a comment about the style.